.github/workflows/build-dry-run.yaml

```
@@ -0,0 +1,19 @@
+name: Build Dry Run
+
+on:
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  Build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Install pnpm
+        uses: pnpm/action-setup@v4
+        with:
+          run_install: true
+          version: 10.10.0
```

在Install pnpm之后，需要进行Install Node.js来启用缓存

```suggestion
      - name: Install pnpm
        uses: pnpm/action-setup@v4
        with:
          run_install: true
          version: 10.10.0
      - name: Install Node.js
        uses: actions/setup-node@v4
        with:
          node-version: 20
          cache: 'pnpm'
```